javascript/modules/client/src/internal/interfaces/context.ts

```diff
@@ -20,7 +16,8 @@ interface VocdoniContextParams {
   // Vocdoni
 }
 interface IpfsContextParams {
-  // ipfs: IpfsOptions;
+  ipfsOptions?: Options;
```

Comment:
Shouldn't this parameter be mandatory within the IPFS scope? Also, shouldn't we allow for receiving N >= 1 nodes info?

Reply:
If not specified, it will default to the lib default parameters that will try to connect to `localhost:5001`. We can overwrite these parameters ones, if not specified, set the default nodes to the aragon ones. About the having more than one node configuration makes sense, but I believe we should add a way to check the state of the IPFS node and In case that it is disconnected shift to the second one and the first selection between the available nodes should be random to avoid connecting always to the same, this does not make sense if Mathias plans to add a load balancer before the nodes.